### PR TITLE
Update board members for Bestuur 54 "Kiwi"

### PR DESCRIPTION
**Pull request by bencarp**

### CODE REVIEW

File: src/locales/nl/pages.yaml [hunk 1]
```diff
@@ -229,15 +229,15 @@ contact:
...
-    Secretaris: Nienke Brouwer ([secretaris@krashna.nl](mailto:secretaris@krashna.nl))\
-    Penningmeester: Mari&euml;tte Sch&ouml;nfeld ([penningmeester@krashna.nl](mailto:penningmeester@krashna.nl))\
-    Acquisiteur en Kamermuziekcommissaris: Rivka Eikelboom ([acquisiteur@krashna.nl](mailto:acquisiteur@krashna.nl) en [kamucom@krashna.nl](mailto:kamucom@krashna.nl))\
-    Orkestcommissaris: Otto Kaaij ([orkestcom@krashna.nl](mailto:orkestcom@krashna.nl))\
-    Koorcommissaris: Anouk de S&eacute;vaux ([koorcom@krashna.nl](mailto:koorcom@krashna.nl))\
-    Publiciteits- en Concertcommissaris: Jochem Schuitemaker ([publicom@krashna.nl](mailto:publicom@krashna.nl) en [concertcom@krashna.nl](mailto:concertcom@krashna.nl))
+    Voorzitter: Marthe Meskers ([voorzitter@krashna.nl](mailto:voorzitter@krashna.nl))\
+    Secretaris: Pim de Jong ([secretaris@krashna.nl](mailto:secretaris@krashna.nl))\
+    Penningmeester: Duschka Kleijn ([penningmeester@krashna.nl](mailto:penningmeester@krashna.nl))\
+    Acquisiteur: Maurice Hooikamer ([acquisiteur@krashna.nl](mailto:acquisiteur@krashna.nl))\
```
Comment (tijshol) [inline src/locales/nl/pages.yaml]:
```suggestion
    Acquisiteur: Maurice Hooikammer ([acquisiteur@krashna.nl](mailto:acquisiteur@krashna.nl))\
```
At least, that's his name on Facebook :)

File: src/locales/en/pages.yaml [hunk 2]
```diff
@@ -157,15 +157,15 @@ contact:
...
-    Secretary: Nienke Brouwer ([secretaris@krashna.nl](mailto:secretaris@krashna.nl))\
-    Treasurer: Mari&euml;tte Sch&ouml;nfeld ([penningmeester@krashna.nl](mailto:penningmeester@krashna.nl))\
-    Acquisition and Chambermusic Commissioner: Rivka Eikelboom ([acquisiteur@krashna.nl](mailto:acquisiteur@krashna.nl) and [kamucom@krashna.nl](mailto:kamucom@krashna.nl))\
-    Orchestra Commissioner: Otto Kaaij ([orkestcom@krashna.nl](mailto:orkestcom@krashna.nl))\
-    Choir Commissioner: Anouk de S&eacute;vaux ([koorcom@krashna.nl](mailto:koorcom@krashna.nl))\
-    Promotion and Concert Commissioner: Jochem Schuitemaker ([publicom@krashna.nl](mailto:publicom@krashna.nl) and [concertcom@krashna.nl](mailto:concertcom@krashna.nl))
+    Chairwoman: Marthe Meskers ([voorzitter@krashna.nl](mailto:voorzitter@krashna.nl))\
+    Secretary: Pim de Jong ([secretaris@krashna.nl](mailto:secretaris@krashna.nl))\
+    Treasurer: Duschka Kleijn ([penningmeester@krashna.nl](mailto:penningmeester@krashna.nl))\
+    Acquisition commissioner: Maurice Hooikamer ([acquisiteur@krashna.nl](mailto:acquisiteur@krashna.nl))\
```
Comment (tijshol) [inline src/locales/en/pages.yaml]:
```suggestion
    Acquisition commissioner: Maurice Hooikammer ([acquisiteur@krashna.nl](mailto:acquisiteur@krashna.nl))\
```
Here as well

**Review by tijshol — CHANGES_REQUESTED** [review]
Two typos, otherwise it's good!

File: src/locales/nl/pages.yaml (hunk 1 above)
Comment (bencarp) [reply src/locales/nl/pages.yaml]: Oh oops. you're absolutely right. Thanks :+1:

**Pull request merged**